start-chrome-for-playr.sh

```diff
@@ -16,14 +16,18 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-# Define the browser to use
-# for Chromium use "/usr/lib/chromium/chromium"
-browser="google-chrome"
-preferences_file="~/.config/google-chrome/Default/Preferences"
+##########################################################################
+#							VARIABLES    								 #
+##########################################################################
+
+# Will be filled with the system identifier will be set during execution
+# Based on OSX or Linux it will be retrieved differently
+system_uuid=""
 
-# Prevent popups or additional empty tabs
-sed -i -E 's/("exited_cleanly":\s*)false/\1true/g' $preferences_file
-sed -i -E 's/("exit_type":\s*)"Crashed"/\1"Normal"/g' $preferences_file
+# Will be filled with the browser path will be set during execution
+# Based on OSX or Linux it will be retrieved differently
+browser=""
+preferences_file="~/.config/google-chrome/Default/Preferences"
```

Comment:
de locatie van deze file is afhankelijk van het OS en of de browser Chrome of Chromium is, we zullen hier dus iets meer logica achter moeten stoppen

Reply:
we still need to make sure the path to preferences file is correct for the OS and browser.

Reply:
We alter the content of the preferences file to prevent error/warning messages that will be shown on the screen (disturbing the signage content) so we need to change the correct file.

Reply:
Preferences file location. Need to check Edge and Vivaldi on Linux rest should be as stated below

Linux
Chromium: ~/.config/chromium/Default/Preferences
Chrome: ~/.config/google-chrome/Default/Preferences
Edge: ?
Brave: ~/.config/BraveSoftware/Brave-Browser/Default/Preferences
Vivaldi: ?


MacOS:
Chromium: ~/Library/Application\ Support/Chromium/Default/Preferences
Chrome: ~/Library/Application\ Support/Google/Chrome/Default/Preferences
Edge: ~/Library/Application\ Support/Microsoft\ Edge/Default/Preferences
Brave: ~/Library/Application\ Support/BraveSoftware/Brave-Browser/Default/Preferences
Vivaldi: ~/Library/Application\ Support/Vivaldi/Default/Preferences